src/sections/blog-post.liquid

```diff
@@ -0,0 +1,155 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is loaded when you add "Blog Posts" section on the Theme Editor.
+
+  We are showing a preview of what the blog post looks like in the preview.
+
+  Required settings
+  - Blog (which blog category to show in your store)
+  - Number of posts to show
+
+  Optional settings
+  - Show author
+  - Show date
+{%- endcomment -%}
+<section>
+  {% if section.settings.title != blank %}
+    <h2>{{ section.settings.title | escape }}</h2>
+  {% endif %}
+
+  {%- comment -%}
+    This refer from blog section when you add the section from Theme Editor
+  {%- endcomment -%}
+  {%- assign blog = blogs[section.settings.blog] -%}
+
+  {% unless blog.articles_count == 0 %}
+    <div class="blog-post">
+      {% for article in blog.articles limit: section.settings.post_limit %}
+        <div class="blog-post__item">
+          {% if article.image %}
+            <a href="{{ article.url }}">
+              {% include 'responsive-image' with
+                  image: article.image,
+                  max_width: 400,
+                  max_height: 400
+              %}
+            </a>
+          {% endif %}
+
+          <div>
+            <h2>
+              <a href="{{ article.url }}">{{ article.title }}</a>
+            </h2>
+
+            {% if section.settings.blog_show_author %}
+              <span>{{ 'blogs.article.by_author' | t: author: article.author }}</span>
+            {% endif %}
+
+            {% if section.settings.blog_show_date %}
+              <span class="blog-post__date">
+                {{ article.published_at | time_tag: format: 'month_day_year' }}
+              </span>
+            {% endif %}
+
+            <div class="blog-post__excerpt">
```

Comment:
Remove this class since it's not needed with changes to CSS mentioned above.

Reply:
I removed.